### ISSUE TITLE
Checking if account verification email has clickable URL

### ISSUE THREAD

**Issue by jivanyatra** [HHMI-HQ/BI-assessment-builder#262]
> Originally opened by @yannisbarlas on 2026-02-20 at 13:25:48 UTC

> **Migrated from GitLab**
> **Original Author:** Christopher Vargas (`@vargasc`)
> **Original Link:** https://gitlab.coko.foundation/coko-org/products/hhmi/hhmi/-/issues/69

The hyperlink in the email provided for account verification wasn’t recognized as a hyperlink to open a browser/new webpage by alt nav technology (dragon naturally speaking) and Voiceover

I believe this may have been corrected in the current version of the account verification email, but I tried to look back through tickets and did not see anything specifically covering this.

**Comment by jivanyatra** [comment]
> Originally commented by @yannisbarlas on 2026-02-20 at 16:47:26 UTC


> **Migrated Comment**
> **Author:** Dione Mentis (`@DioneMentis`)
> **Date:** 2023-03-30T06:15:27.711Z

`@conneelyb`  `@aleezaoshry`  -- We didn't get a confrimation for Chris before he left, but I believe this issue is resolved. Our team cannot replicate it. Can you confirm please
